### PR TITLE
Home 31.01.2019

### PR DESCRIPTION
**Pull request by MateAcademy**

### CODE REVIEW

File: src/main/java/com/mateacademy/lessons12/rest/MateGroupServiceImpl.java [hunk 1]
```diff
@@ -43,4 +43,16 @@ public Response addStudents(@PathParam("groupId") int groupId, Person person) {
 		return Response.status(Status.NOT_FOUND).build();
 	}
 
+	@Override
+	@PUT
+	@Path("/{groupId}/teacher")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response putTeacher(@PathParam("groupId") int groupId, Teacher teacher) {
```
Comment (Taniann) [inline src/main/java/com/mateacademy/lessons12/rest/MateGroupServiceImpl.java:51]: this method doesnt return mateGroup with new teacher, it just returns mateGroup

File: src/main/java/lesson11/soap/MateGroupServiceImpl.java [hunk 2]
```diff
@@ -27,9 +27,6 @@ public MateGroup addStudents(List<Person> persons) {
 		return mateGroup;
 	}
 
-//	@Override
```
Comment (Taniann) [inline src/main/java/lesson11/soap/MateGroupServiceImpl.java:30]: better dont add redundant code :)

**Review by Taniann — CHANGES_REQUESTED** [review]
read comments
and the screenshots you've added are without using new methods

File: src/main/java/com/mateacademy/lessons12/rest/MateGroupServiceImpl.java (hunk 1 above)
Comment (SerhiiPasko) [reply src/main/java/com/mateacademy/lessons12/rest/MateGroupServiceImpl.java:51]: Yes, you dont use teacher anything

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments

**Pull request closed without merging**